lib/rules.ex

```diff
@@ -0,0 +1,31 @@
+defmodule Rules do
+  def winning_combos(board) do
+    Enum.into(Board.row(board), Board.column(board))
+    |> Enum.into(Board.diagonal(board))
```

Comment:
Consider breaking up that first line:

Board.row(board)
|> Enum.into(Board.column(board))
|> Enum.into(Board.diagonal(board))